nbgrader/html/templates/notebook_submissions.tpl

```diff
@@ -28,7 +28,9 @@
 <tbody>
   {%- for submission in submissions -%}
   <tr>
-    <td><a href="/submissions/{{ submission.id }}">Submission #{{ submission.index + 1 }}</a></td>
+    <td>{%- if notebook_server_exists -%}<a href="/submissions/{{ submission.id }}">{%- endif -%}
```

Comment:
I don't think you want the if statement here -- otherwise this prevents people from doing any grading. The part that depends on the live notebook is in `formgrade.tpl` here: https://github.com/jupyter/nbgrader/blob/master/nbgrader/html/templates/formgrade.tpl#L65